src/Extensions/MemberExtension.php

```diff
@@ -68,4 +64,17 @@ public function getJWTData()
 
         return Convert::raw2json($data);
     }
+
+    /**
+     * Destroy all JWT tokens
+     *
+     * @return Member
+     */
+    public function DestroyAuthTokens(): Member
```

Comment:
Not PSR compliant to start with a capital

Reply:
That was authully careless of me.